elements/pfe-button/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@patternfly/pfe-button",
+  "description": "Button element for PatternFly Elements",
+  "pfelement": {
+    "className": "PfeButton",
+    "elementName": "pfe-button",
+    "files": {
+      "styles": "./src/pfe-button.scss",
+      "template": "./src/pfe-button.html",
+      "js": "./src/pfe-button.js"
+    },
+    "assets": []
+  },
+  "version": "1.0.0-prerelease.39",
+  "keywords": [
+    "web-components",
+    "html"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "github:patternfly/patternfly-elements",
+    "directory": "elements/pfe-button"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "./dist/pfe-button.umd.js",
+  "module": "./dist/pfe-button.js",
+  "unpkg": "./dist/pfe-button.umd.min.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "../../node_modules/.bin/gulp && ../../node_modules/.bin/prettier --ignore-path ../../.prettierignore --write '**/*.{js,json}'",
+    "dev": "../../node_modules/.bin/gulp dev",
+    "watch": "../../node_modules/.bin/gulp watch",
+    "test": "../../node_modules/.bin/wct --configFile ../../wct.conf.json elements/pfe-button/test/"
+  },
+  "contributors": [
+    {
+      "name": "Kyle Buchanan"
```

Comment:
@eyevana and @wesruv would you like to list yourselves here too?

Reply:
I'm fine with being left out, I just griped about code comments :smile: